D4e/d4e_functions.py

```diff
@@ -349,26 +349,16 @@ async def callback(self, interaction: discord.Interaction):
                     output_string = await saveIndependent(
                         self.engine, self.bot, self.guild, self.character.name, self.condition.title, modifier=""
                     )
-                await interaction.edit_original_message(content=output_string)
-                # await interaction.response.send_message(
-                #     await save(
-                #       self.ctx,
-                #       self.engine,
-                #       self.bot,
-                #       self.character.name,
-                #       self.condition.title,
-                #       modifier=""
-                #     )
-                # )
+                await interaction.edit_original_response(content=output_string)
                 await initiative.block_update_init(
                     self.ctx, interaction.message.id, self.engine, self.bot, guild=self.guild
                 )
             except Exception:
                 output_string = "Unable to process save, perhaps the condition was removed."
-                await interaction.edit_original_message(content=output_string)
+                await interaction.edit_original_response(content=output_string)
```

Comment:
🎉

Reply:
I got a warning that edit_original_message was being depreciated for edit_original_response, so I just made the changes